namespace java thrift.test

struct myI16Struct {
  1: i16 a
}
